lib/mbaasRequest/mbaasRequest.js

```diff
@@ -175,6 +175,15 @@ function responseHandler(params, cb) {
 
     if (err || (httpResponse && httpResponse.statusCode >= 400)) {
 
+      // MbaaS is unreachable
+      if (err.code !== undefined && err.code === "ENOTFOUND" || err.code === "ETIMEDOUT") {
+        log.logger.error("Error : Status Code", err.code, err);
+        err = {
+          httpCode: 500,
+          message: "MBaaS environment is not reachable. Please make the environment available or delete it, and then retry the project delete."
```

Comment:
Can't have a message specifically for project delete in this function as it handles all of the requests made to an mbaas (e.g. list a form, deploy a form.)